libs/coin-modules/coin-aptos/src/test/bot-specs.ts

```diff
@@ -17,9 +18,11 @@ const aptosSpecs: AppSpec<Transaction> = {
   appQuery: {
     model: DeviceModelId.nanoSP,
     appName: "Aptos",
+    appVersion: "0.8.11",
+    firmware: "1.3.2",
```

Comment:
better to remove this from here, the bot should work with whatever the engine finds as being latest